src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1273,7 +1273,9 @@ internal enum ErrorCode
         ERR_BadPdbData = 8036,
         // available 8037-8039
         INF_UnableToLoadSomeTypesInAnalyzer = 8040,
-        // available 8041-8049
+        WRN_LoadedAnalyzerDiffers = 8041,
+        WRN_MissingAnalyzerDependency = 8042,
```

Comment:
@shyamnamboodiripad why would we make them non-configurable? I might want to turn them into errors or suppress them if I don't care about analyzers getting skipped for a build.